### DISCUSSION TITLE
Cant deploy to a live network

### DISCUSSION THREAD

**Discussion by 203kingzzley** [smartcontractkit/full-blockchain-solidity-course-py#1636] [Q&A] [answered]
After trying to connect my metamask wallet to remix so I can deploy it the address of my wallet doesn't show on remix. I need an answer so I can progress.

**Comment by cromewar** [comment, marked as answer]
Hello @203kingzzley 
Your question is not clear, please take a look to #980 and reformulate your question.
By far it seems you are not selecting "injected web3" on REMIX.
Cheers.

**Reply by 203kingzzley** [reply]
Sorry for not clarifying my question, I'm just saying that after connecting my metamask wallet to the remix  ide the address of my wallet isn't showing on remix and also if I click on injected web3 after connecting it ends up changing back to remix vm
Here's a picture of what I'm talking about 
![Screenshot (43)](https://user-images.githubusercontent.com/101167154/184235474-21c0289e-d16e-4cf7-8785-20ed4c43a20e.png)

**Reply by 203kingzzley** [reply]
Please I really need help solving this issue it's hindering my learning

**Reply by cromewar** [reply]
I would said the problem is related to the browser itself, please try at least one of this steps and see if they works:
1. Delete all the cookies, cacha and history of your browser (create a backup of you code because REMIX will not remember it).
2. Try another browser (not Chromium based) like Friefox o Opera, does the problem persists?

I hope this info might help.

**Reply by 203kingzzley** [reply]
Thanks I tried deleting all the cookies and clearing the cache  and it's working perfectly now, thanks a lot

**Reply by cromewar** [reply]
I'm glad your problem is solved now.
